config/initializers/devise_token_auth.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+DeviseTokenAuth.setup do |config|
+  # By default the authorization headers will change after each request. The
+  # client is responsible for keeping track of the changing tokens. Change
+  # this to false to prevent the Authorization header from changing after
+  # each request.
+  config.change_headers_on_each_request = false
```

Comment:
Why are you changing this from default?

Reply:
I read somewhere on stack overflow that it would be better for testing purposes, but I will change it back to true :D